ops/environments/000_cross_env_vars

```diff
@@ -0,0 +1,139 @@
+# Anistrano
+ansistrano_deploy_from: "{{ playbook_dir }}/../" # Where my local project is (relative or absolute path)
+ansistrano_deploy_to: "/opt/{{ project_name }}" # Base path to deploy to.
+ansistrano_version_dir: "releases" # Releases folder name
+ansistrano_current_dir: "current" # Softlink name. You should rarely changed it.
+ansistrano_current_via: "symlink"
+ansistrano_keep_releases: 3 # Releases to keep after a new deployment. See "Pruning old releases".
+ansistrano_shared_paths:
+  - log
+  - public/assets
+  - public/system
+  - tmp/cache
+  - tmp/derivatives
+  - tmp/pids
+  - tmp/sockets
+  - tmp/uploads
+  - vendor/bundle
+
+ansistrano_shared_files:
+  - config/database.yml
+  - config/secrets.yml
+  - config/fedora.yml
+  - config/blacklight.yml
+  - config/redis.yml
+  - config/resque-pool.yml
+  - config/solr.yml
+
+ansistrano_deploy_via: "git"
+#ansistrano_git_repo: https://github.com/ucsdlib/horton.git
+ansistrano_git_repo: https://github.com/notch8/horton.git
```

Comment:
This is temporary still just for testing I assume?

Reply:
correct